scrapi/harvesters/umontreal.py

```diff
@@ -7,14 +7,31 @@
 '''
 from __future__ import unicode_literals
 
+from scrapi.base.helpers import updated_schema
 from scrapi.base import OAIHarvester
 
 
+def umontreal_language_processor(languages):
+
+    if not languages:
+        languages = []
+
+    return languages
```

Comment:
In the original schema, each item in the languages array is transformed to a ISO 639-3 code, for example, 'English' becomes 'eng' and 'Russian' becomes 'rus'. I checked the source and it seems that they were not correctly coded (for example it uses 'en' instead of 'eng'). There is a function get_code in helpers.py that can do that for you. Can you implement it in your function to return the correct codes?